samples/bluetooth/peripheral_proximity/README.rst

```diff
@@ -0,0 +1,81 @@
+.. _peripheral_Proximity:
+
+Bluetooth: Peripheral Proximity
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Overview
+********
+
+The Proximity Application is an example that implements the Proximity Profile.The Proximity Profile alerts the user when connected devices are too far apart.
+The application includes the following services:
+
+For the Proximity Profile:
+Link Loss Service
+Immediate Alert Service
+TX Power Service
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires a phone or tablet running a compatible application, for example `nRF Connect for Mobile`_ or `nRF Toolbox`_.
+
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_proximity`
+
+.. include:: /includes/build_and_run.txt
+
+.. _peripheral_proximity_testing:
+
+Testing
+=======
+
+After programming the sample to your dongle or development kit, test it by performing the following steps:
+
+1. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. Power on the development kit.
+#. Connect to the device from the nRF Connect application.
+   The device is advertising as ``Nordic_Proximity``.
+   The services of the connected device are shown.
+#. In :guilabel:` Immediate Alert Service`, Set the AlertLevel characteristic in Immediate Alert Service to 1 (enter '01' in the text box, and click the 'write' button), and observe the Mild alert(LED 1   is blinking (period 1200 msec, duty cycle: 50%)) .
```

Comment:
```suggestion
#. In :guilabel:`Immediate Alert Service`, set the AlertLevel characteristic to ``1`` (enter ``01`` in the text box, and click :guilabel:`Write`), and observe the Mild alert (**LED 1** is blinking (period 1200 msec, duty cycle: 50%)) .
```
Not sure whether the write button is capitalized or not. Check this to verify it.